libraries/splunk_app.rb

```diff
@@ -266,6 +273,18 @@ def install_from_tar(filename, expected_version, installed_version)
         tarfile.close if tarfile
       end
 
+      def manage_lookups
+        lookups = new_resource.lookups
+        lookups.each do |file_name, url|
+          if url && !url.empty?
+            fail "Unsupported lookup file format for #{file_name} in the app #{new_resource.app}" unless file_name =~ /\.(?:csv.gz|csv|kmz)$/i
```

Comment:
Addressed in this commit: https://github.com/nmattam/cerner_splunk/commit/87fd4fb5ccacd0fdf083317dd0804c8f913e747e

Commit still not showing up in the PR due to some issues with github.